site/static/examples/sample-validating-policy.yaml

```diff
@@ -15,5 +15,5 @@ spec:
       operations:  ["CREATE", "UPDATE"]
       resources:   ["jobsets"]
   validations:
-    - expression: "'kueue.x-k8s.io/queue-name' in object.metadata.labels && object.metadata.labels['kueue.x-k8s.io/queue-name'] != ''"
+    - expression: "has(object.metadata.labels) && 'kueue.x-k8s.io/queue-name' in object.metadata.labels && object.metadata.labels['kueue.x-k8s.io/queue-name'] != ''"
```

Comment:
Avoid getting a lower-level error message when object does not have labels.
Without the extra check you get:
```
The jobs "sample-job" is invalid: : ValidatingAdmissionPolicy 'mlbatch-require-queue-name' with binding 'mlbatch-require-queue-name' denied request: expression ''kueue.x-k8s.io/queue-name' in object.metadata.labels && object.metadata.labels['kueue.x-k8s.io/queue-name'] != ''' resulted in error: no such key: labels
```